Tests/Unit/NetworkServer/CupsProtocolMessageProcessorTests.cs

```diff
@@ -41,7 +41,9 @@ public CupsProtocolMessageProcessorTests()
             this.logger = new Mock<ILogger<CupsProtocolMessageProcessor>>();
             this.processor = new CupsProtocolMessageProcessor(this.basicsStationConfigurationService.Object,
                                                               this.deviceAPIServiceBase.Object,
-                                                              this.logger.Object);
+                                                              this.logger.Object,
+                                                              // Do not pass meter since metric testing will be unreliable due to interference from test classes running in parallel.
+                                                              null);
```

Comment:
[nit] we could also pass a `TestMeter.Instance` if you don't like the nullability here

Reply:
I like nullability ❤️